src/shared/ask-password-api.c

```diff
@@ -421,7 +421,7 @@ int ask_password_tty(
         if (!message)
                 message = "Password:";
 
-        if (emoji_enabled())
+        if ((flags & ASK_PASSWORD_NO_EMOJI) && emoji_enabled())
```

Comment:
Maybe, the condition is inverted, i.e. missing `!`?
I'd like to inverse the meaning of the flag: `ASK_PASSWORD_NO_EMOJI` -> `ASK_PASSWORD_USE_EMOJI` or so.

Also, not a matter, but I prefer to use `FLAGS_SET()`.

Reply:
Yes, I had the complete logic inverted... It was too late yesterday. 😜

However I kept `ASK_PASSWORD_NO_EMOJI` (with fixed logic). Showing the emoji is the default, imho the flag should be set for non-default behavior.